client/my-sites/plugins/plugin-details.jsx

```diff
@@ -316,12 +316,12 @@ function PluginDetails( props ) {
 							{ translate( 'Plugin details' ) }
 						</div>
 						<div className="plugin-details__plugin-details-content">
-							<div className="plugin-details__downloads">
-								<div className="plugin-details__downloads-text title">
-									{ translate( 'Downloads' ) }
+							<div className="plugin-details__active-installs">
+								<div className="plugin-details__active-installs-text title">
+									{ translate( 'Active installations' ) }
 								</div>
-								<div className="plugin-details__downloads-value value">
-									{ formatNumberCompact( fullPlugin.downloaded, 'en' ) }
+								<div className="plugin-details__active-installs-value value">
+									{ formatNumberCompact( fullPlugin.active_installs, 'en' ) }
```

Comment:
Eg. Woocommerce have ~5M installs, but it's a bit hard to understand at a first glance, because it is formatted as 5000K:
![image](https://user-images.githubusercontent.com/11555574/142389631-5193874d-0bd0-4879-b1a9-614bbd06e59e.png)

We could use `formatNumberMetric` instead of `formatNumberCompact` to format the active installs with more decimals: https://github.com/Automattic/wp-calypso/blob/31d3844d216c261ed9bff5ea0abadc0aa0d78b4b/client/lib/format-number-compact/index.js#L48

Reply:
Great catch @gcsecsey! I used `formatNumberMetric` as you recommended 
![SS 2021-11-18 at 12 01 08](https://user-images.githubusercontent.com/12430020/142393993-5f551f71-1d2e-456f-a72d-491c101273c8.png)